packages/ontario-frontend-cli/core/questions/ontario-remove-package/removePackageQuestion.js

```diff
@@ -0,0 +1,14 @@
+const { question } = require('../../utils/styling/textStyling');
+
+const removePackageQuestion = (cmd) => {
+  return {
+    type: 'list',
+    name: 'removePackage',
+    message: question(`Are you sure you want to remove ${cmd} and it\'s associated packages and configuration files?`),
```

Comment:
```suggestion
    message: question(`Are you sure you want to remove ${cmd} and its associated packages and configuration files?`),
```

Reply:
I think the `'s` is the correct grammar structure here but I've gone and removed the escaping `\`.